internal/cmd/login/command_test.go

```diff
@@ -832,9 +832,8 @@ func TestLoginWithExistingContext(t *testing.T) {
 		loginCmd, cfg := newLoginCmd(auth, userInterface, s.isCloud, req, mockNetrcHandler, AuthTokenHandler, mockLoginCredentialsManager, LoginOrganizationManager)
 
 		// Login to the CLI control plane
-		output, err := pcmd.ExecuteCommand(loginCmd, s.args...)
+		_, err := pcmd.ExecuteCommand(loginCmd, s.args...)
 		req.NoError(err)
-		req.NotContains(output, fmt.Sprintf(errors.LoggedInAsMsg, promptUser))
```

Comment:
Is this unnecessary because it's already caught by integration tests?

Reply:
Yep!